crates/core/src/masp.rs

```diff
@@ -104,7 +104,7 @@ impl Display for MaspTxId {
     Serialize,
     Deserialize,
 )]
-pub struct MaspEpoch(Epoch);
+pub struct MaspEpoch(pub Epoch);
```

Comment:
This has been made public to extract the inner `Epoch` when constructing the refund storage key. Can't we instead implement `KeySeg` on `MaspEpoch`? Otherwise could we come up with a getter to extract the inner epoch from here? I'd like to avoid exposing the inner type as the whole point of this type was to ensure safe operations on it